TAScheduler/tests/AcceptanceTests/createLab.py

```diff
@@ -0,0 +1,118 @@
+                         "An error message was not displayed when duplicate Lab was created")
+
+    def test_invalidHours(self):
+        r = self.dummyClient.post('/create_lab/', {"labID": 2, "labName": "Lab", "labLocation": "EMS 280",
+                                                   "labHours": "invalid input", "days": "M, W",
+                                                   "course": self.course, "TA": self.TA})
+        self.assertEqual(r.context["error"],
+                         "Lab section was not created. invalid literal for int() with base 10: 'Software Engineering'",
+                         "An error message was not displayed when duplicate Lab was created")
+
+    def test_blankDays(self):
+        r = self.dummyClient.post('/create_lab/', {"labID": 2, "labName": "Lab", "labLocation": "EMS 280",
+                                                   "labHours": "03:00 PM - 04:00 PM", "days": "",
+                                                   "course": self.course, "TA": self.TA})
+        self.assertEqual(r.context["error"],
+                         "Lab section was not created. invalid literal for int() with base 10: 'Software Engineering'",
+                         "An error message was not displayed when duplicate Lab was created")
+
+    def test_invalidDays(self):
+        r = self.dummyClient.post('/create_lab/', {"labID": 2, "labName": "Lab", "labLocation": "EMS 280",
+                                                   "labHours": "03:00 PM - 04:00 PM", "days": "invalid input",
+                                                   "course": self.course, "TA": self.TA})
+        self.assertEqual(r.context["error"],
+                         "Lab section was not created. invalid literal for int() with base 10: 'Software Engineering'",
+                         "An error message was not displayed when duplicate Lab was created")
+
+    def test_courseNotExisting(self):
+        r = self.dummyClient.post('/create_lab/', {"labID": 2, "labName": "Lab", "labLocation": "EMS 280",
+                                                   "labHours": "03:00 PM - 04:00 PM", "days": "M, W",
+                                                   "course": "", "TA": self.TA})
+        self.assertEqual(r.context["error"],
+                         "Lab section was not created. invalid literal for int() with base 10: ''",
+                         "An error message was not displayed when duplicate Lab was created")
+
+    def test_createLab(self):
+        r = self.dummyClient.post('/create_lab/', {"labID": 10, "labName": "Lab2", "labLocation": "EMS 280",
+                                                   "labHours": "03:00 PM - 04:00 PM", "days": "M, W",
+                                                   "course": self.course2, "TA": self.TA})
+        self.assertEqual(Lab.objects.get(labID=1).name, "Lab", "Lab was created successfully")
```

Comment:
```suggestion
        self.assertEqual(Lab.objects.get(labID=1), Lab(labID=10, name="Lab2", location="EMS 280", hours=03:00 PM - 04:00 PM", days="M, W", course=self.course2, TA=self.TA, "Lab was not created successfully")
```

Reply:
Also I think this one should be assertNotEqual correct me if I'm wrong